.gitignore

```diff
@@ -14,3 +14,6 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
You're adding this to the gitignore, but it's already checked into source control. If intent is to actually ignore it, will also need to `git rm`